miqcli/api.py

```diff
@@ -189,7 +190,7 @@ def _generate_token(self):
         the client will exit with -1.
         """
         if self._username is None or self._password is None:
-            print('You need to set username and password.')
+            log.error('You need to set username and password.')
             exit(-1)
```

Comment:
Maybe you add `abort=True` to the `log.error` and remove the line below with `exit(-1)` ?

Reply:
I was contemplating your suggestion when committing. I agree, I think we should. This way its all handled in one line. I will make the change and submit a new commit.